### PR TITLE
Update templates handling #LMR-1685

### PR DESCRIPTION
**Pull request by marvec**

### CODE REVIEW

File: src/app/view/perspectives/gantt-chart/util/gantt-chart-util.ts [hunk 1]
```diff
@@ -192,7 +192,8 @@ export function isGanttConfigChanged(viewConfig: GanttChartConfig, currentConfig
     return true;
   }
 
-  return ganttConfigCollectionsChanged(viewConfig.collections || {}, currentConfig.collections || {});
+  const isChanged = ganttConfigCollectionsChanged(viewConfig.collections || {}, currentConfig.collections || {});
+  return isChanged;
```
Comment (kubedo8) [inline src/app/view/perspectives/gantt-chart/util/gantt-chart-util.ts]: I think you can revert these changes back.

Comment (marvec) [reply src/app/view/perspectives/gantt-chart/util/gantt-chart-util.ts]: Correct, this was only for debugging purposes!
